scripts/master_voting/example_votes.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
These votes are valid over `example_proposal.json` but only when the on-chain voting round is `0`.